src/layouts/Default.tsx

```diff
@@ -43,37 +43,39 @@ export function Default (props: PropsWithChildren<ScanAppProps>): JSX.Element |
         <link rel='icon' type='image/png' sizes='48x48' href='/favicon.png' />
       </Head>
 
-      <NetworkProvider>
-        <WhaleProvider>
-          <StoreProvider state={props.initialReduxState}>
-            <StatsProvider>
-              <SupplyProvider>
-                <PoolPairsProvider>
-                  <WarningBanner mainnet>
-                    <div>
-                      There is a display issue where dAMZN-related values are incorrectly displayed. Funds are safe on
-                      the blockchain. Read the
-                      <a
-                        href='https://github.com/JellyfishSDK/jellyfish/issues/1532'
-                        className='text-primary-500 hover:text-primary-600 font-medium'
-                        target='_blank' rel='noreferrer'
-                      >
-                        <span> GitHub issue </span>
-                      </a>
-                      for more information.
-                    </div>
-                  </WarningBanner>
-                  <Header />
-                  <main className='flex-grow'>
-                    {props.children}
-                  </main>
-                  <Footer />
-                </PoolPairsProvider>
-              </SupplyProvider>
-            </StatsProvider>
-          </StoreProvider>
-        </WhaleProvider>
-      </NetworkProvider>
+      <ThemeProvider theme='light'>
+        <NetworkProvider>
```

Comment:
Another thing is, do we want this to be the first layer of provider? Not sure if the other providers depend on this, could be moved to the inner part

Reply:
updated